tools/jil/server/asset-server.js

```diff
@@ -112,6 +112,10 @@ class AgentInjectorTransform extends AssetTransform {
     return updatedConfig
   }
 
+  getAjaxDenyListString(){
+    return runnerArgs.denyListBam ? `window.NREUM||(NREUM={init:{}});NREUM.init.ajax=NREUM.init.ajax||{};NREUM.init.ajax.deny_list=NREUM.init.ajax.deny_list||[];NREUM.init.ajax.deny_list.push('bam-test-1.nr-local.net')` : ''
```

Comment:
Might want to put a `;` before and after the entire string for sanity. Idk if initContent necessarily has that